### ISSUE TITLE
uf_plotParam() does not auto-run uf_predictContinuous(), input cfg.predictAt ignored

### ISSUE THREAD

**Issue by olafdimigen** [unfoldtoolbox/unfold#44]
Let's say I want to plot a continuous predictor at three specified levels:

ufresult = uf_condense(EEG);
cfg = [ ];
cfg.channel = 1;
cfg.predictAt = {{'sac_amplitude',[0.5 1.5 3]}};
cfg.add_intercept = 1;
uf_plotParam(ufresult,cfg); % --> **WRONG PLOT OUTPUT**: plotted lines are NOT at 3 values provided 

cfg2 = [ ];
cfg2.cfg2.predictAt = {{'sac_amplitude',[0.5 1.5 3]}}
ufresult2 = uf_predictContinuous(ufresult, cfg2)
uf_plotParam(ufresult,cfg); --> **CORRECT PLOT OUTPUT**, lines at 0.5 1.5 and 3

Turns out that the input cfg.predictAt is (virtually) ignored in uf_plotParam

**Comment by olafdimigen** [comment]
Yes, the if is executed, but the the uf_predictContinuous() function simply never "saw" the predicAt values since it was called like this:

ufresult = uf_predictContinuous(ufresult,'deconv',cfg.deconv);  % no values given, they are not in ufresult

Committed a change that should fix the issue:

ufresult = uf_predictContinuous(ufresult,'deconv',cfg.deconv,'predictAt',cfg.predictAt);

**Comment by behinger** [comment]
cool! thanks - we can close this then?
